app/src/lib/hostedServices/github/githubPrService.test.ts

```diff
@@ -0,0 +1,28 @@
+import { GitHubPrService } from './githubPrService';
+import { Octokit, type RestEndpointMethodTypes } from '@octokit/rest';
+import { expect, test, describe, vi } from 'vitest';
+
+// TODO: Rewrite this proof-of-concept into something valuable.
+describe.concurrent('GitHubPrService', () => {
+	test('Test parsing response', async () => {
+		const title = 'PR Title';
+		const octokit = new Octokit();
+		vi.spyOn(octokit.pulls, 'get').mockReturnValueOnce(
+			Promise.resolve({
+				data: { title }
+			} as RestEndpointMethodTypes['pulls']['get']['response'])
+		);
+		const service = new GitHubPrService(
+			octokit,
+			{
+				provider: 'github.com',
+				name: 'test-repo',
+				owner: 'test-owner'
+			},
+			'base-branch',
+			'upstream-branch'
+		);
+		const pr = await service.get(123);
+		expect(pr?.title).equal(title);
+	});
+});
```

Comment:
@Caleb-T-Owens @ndom91 just fyi. I will still add a more meaty test for the pr monitor before merging  anything.